utilities/pipelines/staticValidation/module.tests.ps1

```diff
@@ -342,7 +342,13 @@ Describe 'Readme tests' -Tag Readme {
                 $expectedColumnsInOrder += @('Description')
 
                 $readMeCategoryIndex = $readMeContent | Select-String -Pattern "^\*\*$paramCategory parameters\*\*$" | ForEach-Object { $_.LineNumber }
-                $readmeCategoryColumns = ($readMeContent[$readMeCategoryIndex] -split '\|') | ForEach-Object { $_.Trim() } | Where-Object { -not [String]::IsNullOrEmpty($_) }
+
```

Comment:
Could you elaborate on this change? Is it related to the metric alert testing update?

Reply:
Hey @eriqua - I'm not sure what the metric alert test is - but Marius introduced a change to the ReadMe gen in main as part of his workstream and which led to this test failing. @ahmadabdalla just introduced a hotfix and regenerated all ReadMe's (#2132) - and this was essentially my fix. I wasn't sure where the error came from and hence fixed it only for the PRs I worked on in the past week. Will open a dedicated PR now.

Reply:
Created PR #2133

Reply:
This should then also take care of the conflict

Reply:
Thanks for opening a dedicated PR. Merge conflicts were unfortunately introduced after its merge and should be solved.

Reply:
done